network.go

```diff
@@ -479,6 +485,17 @@ func NetworkOptionPersist(persist bool) NetworkOption {
 	}
 }
 
+// NetworkOptionIsolatedNetwork returns an option setter to set if a network should be isolated
+func NetworkOptionIsolatedNetwork(isolated bool) NetworkOption {
```

Comment:
Given the default is that libnetwork created networks are not isolated, can we change the method to not take any bool: `NetworkOptionIsolatedNetwork()`. 

Besides this, the changes LGTM.
Thanks